### PR TITLE
Kaw 7680 footer

### PR DESCRIPTION
**Pull request by TomaszDziezykNetcentric**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #KAW-7680

Test URLs:
- Before: https://main--edge-delivery-bimota--netcentric.hlx.page/drafts/tdziezyk/
- After: https://kaw-7680-footer--edge-delivery-bimota--netcentric.hlx.page/drafts/tdziezyk/

### CODE REVIEW

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -1,14 +1,173 @@
 footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
+  background-color: #222;
+  color: #fff;
```
Comment (lakshmishriaswathnarayana) [inline blocks/footer/footer.css]: Can you use the variable here also in other places in this file ?

File: blocks/footer/footer.js [hunk 2]
```diff
@@ -16,5 +17,61 @@ export default async function decorate(block) {
...
+  const backToTopIcon = `
+    <svg width="18" height="10" viewBox="0 0 18 10">
+      <polyline fill="none" stroke="currentColor" stroke-width="1.2" points="1 9 9 1 17 9"></polyline>
+    </svg>
+  `;
+
+  const backToTopText = getTextLabel('top');
+
+  const backToTopNode = document.createRange().createContextualFragment(`
+    <button id="back-to-top">
```
Comment (lakshmishriaswathnarayana) [inline blocks/footer/footer.js]: Can you use classname instead here ?

**Review by lakshmishriaswathnarayana — CHANGES_REQUESTED** [review]

File: blocks/footer/footer.css (hunk 1 above)
Comment (TomaszDziezykNetcentric) [reply blocks/footer/footer.css]: Done

File: blocks/footer/footer.js (hunk 2 above)
Comment (TomaszDziezykNetcentric) [reply blocks/footer/footer.js]: Done

**Review by lakshmishriaswathnarayana — APPROVED** [review]

**Pull request merged**